src/website/views/components/toolbar/_properties.pug

```diff
@@ -1,6 +1,170 @@
 h2 Web Component
-h3 Properties
-.chi-card.-align--center.-empty
-  .chi-card__content
-    .chi-card__caption
-      | Web component coming soon
+:stencilDoc(name='chi-toolbar')
+
+.chi-divider.-my--5
+
+h2 Chi Vue
+h3 Props
+
+h4 Search Input Props
+p.-text
+ | To see props for the <code>search input</code> you must visit its documentation <a href="../forms/search-input/#props">here</a>
+
+h4 Data Table Filters Props
+section.chi-table.chi-table__options.-bordered.-my--3
+  div(style='overflow-x:auto;')
+    table.-text(cellpadding='0' cellspacing='0')
+      thead
+        tr
+          th
+            div Property
+          th
+            div Description
+          th
+            div Type
+          th
+            div Default
+      tbody
+        tr
+          td
+            div
+              code filtersData
+          td
+            div To define filters in the toolbar.
+          td
+            div
+              code DataTableFilters[]
+          td
+            div
+              code []
+        tr
+          td
+            div
+              code portal
+          td
+            div To define portal style of the toolbar.
+          td
+            div
+              code DataTableFilters[]
+          td
+            div
+              code []
+
+h4 Column Customization Props
+section.chi-table.chi-table__options.-bordered.-my--3
+  div(style='overflow-x:auto;')
+    table.-text(cellpadding='0' cellspacing='0')
+      thead
+        tr
+          th
+            div Property
+          th
+            div Description
+          th
+            div Type
+          th
+            div Default
+      tbody
+        tr
+          td
+            div
+              code columnsData
+          td
+            div To define columns for customization.
+          td
+            div
+              code DataTableColumns[]
+          td
+            div
+              code []
+
+h3 Interfaces and types
+section.chi-table.chi-table__options.-bordered.-my--3
+  div(style='overflow-x:auto;')
+    table.-text(cellpadding='0' cellspacing='0')
+      thead
+        tr
+          th
+            div Name
+          th
+            div Interface
+      tbody
+        tr
+          td
+            div
+              code DataTableFilter
+          td
+            div 
+              :code(lang='html')
+                interface DataTableFilter {
+                  name: string;
+                  label: string;
```

Comment:
```suggestion
                  label: <code>string</code>;
```
Do the same for the rest of types, please

Reply:
Solved.